game/src/ecs/components/skill/FireballSkill.java

```diff
@@ -10,7 +10,7 @@ public FireballSkill(ITargetSelection targetSelection) {
                 "skills/fireball/fireBall_Down/",
                 0.5f,
                 new Damage(1, DamageType.FIRE, null),
-                new Point(10, 10),
+                new Point(1, 1),
```

Comment:
Die Änderung verstehe ich grad nicht so richtig, die scheint keinen Zusammenhang mit der Entity-Sache zu haben? Ist das ein Versehen?

Reply:
Jetzt bin ich in die selbe Mentale Fehle gefallen wie die SHKs....

Ja streng genommen gehört das nicht hier her....

Über ecken könnte ich jetzt sagen, gehört ja zum Hero..

Reply:
Ja, wegen mir. Aber ich verstehe die Änderung nicht ...

Reply:
Inhaltlich: 
Die Hitbox war zu mit 10x10 Tiles zu groß. 1x1 Tile reicht vollkommen.